pkg/agent/util/net.go

```diff
@@ -180,6 +182,57 @@ func GetIPNetDeviceByName(ifaceName string) (v4IPNet *net.IPNet, v6IPNet *net.IP
 	return nil, nil, nil, fmt.Errorf("unable to find local IP and device")
 }
 
+func GetIPNetDeviceByCIDRs(cidrsList string) (v4IPNet, v6IPNet *net.IPNet, link *net.Interface, err error) {
+	cidrs, err := utilnet.ParseCIDRs(strings.Split(strings.TrimSpace(cidrsList), ","))
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	dualStack, err := utilnet.IsDualStackCIDRs(cidrs)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	if len(cidrs) > 1 && !dualStack {
+		return nil, nil, nil, fmt.Errorf("len of cidrs is:%v and they are not configured as dual stack (at least one from each IPFamily)", len(cidrs))
```

Comment:
```suggestion
		return nil, nil, nil, fmt.Errorf("len of cidrs is %v and they are not configured as dual stack (at least one from each IPFamily)", len(cidrs))
```
Otherwise the format would be a little weird: `len of cidrs is:2 and ...`, normally a space should be added after `:`

Reply:
Updated